offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/create/CreateProductViewModel.groovy

```diff
@@ -0,0 +1,27 @@
+package life.qbic.portal.offermanager.components.product.create
+
+import groovy.beans.Bindable
+import life.qbic.datamodel.dtos.business.ProductCategory
+import life.qbic.datamodel.dtos.business.services.ProductUnit
+
+
+/**
+ * <h1>Holds all values that the user specifies in the CreateProductView</h1>
+ *
+ * @since 1.0.0
+ *
+*/
+class CreateProductViewModel {
+
+    @Bindable String productName
+    @Bindable Boolean productNameValid
+    @Bindable String productDescription
+    @Bindable Boolean productDescriptionValid
+    @Bindable String productUnitPrice
```

Comment:
Don't forget that the Price has to be cast as a double when provided to the Use Case later.